DCHdigi/src/DCHdigi_v01.cpp

```diff
@@ -99,8 +97,23 @@ StatusCode DCHdigi_v01::initialize() {
 std::tuple<extension::SenseWireHitCollection, extension::SenseWireHitSimTrackerHitLinkCollection>
 DCHdigi_v01::operator()(const edm4hep::SimTrackerHitCollection& input_sim_hits,
                     const edm4hep::EventHeaderCollection&   headers) const {
+  /// initialize engines
+  std::mt19937_64 m_engine;
```

Comment:
```suggestion
  std::mt19937_64 rng_engine;
```
Prefix `m_` is probably reserved for members

Reply:
thanks! changed!